// [[Reason: Fixing enum values is not supported]] ////

//// [tests/cases/conformance/constEnums/constEnum2.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -8,16 +8,20 @@
     f,
     g
 }
 //# sourceMappingURL=constEnum2.d.ts.map
+
 /// [Errors] ////
 
+constEnum2.ts(10,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 constEnum2.ts(10,9): error TS2474: const enum member initializers must be constant expressions.
+constEnum2.ts(11,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 constEnum2.ts(11,9): error TS2474: const enum member initializers must be constant expressions.
+constEnum2.ts(12,5): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 constEnum2.ts(12,9): error TS2474: const enum member initializers must be constant expressions.
 
 
-==== constEnum2.ts (3 errors) ====
+==== constEnum2.ts (6 errors) ====
     // An enum declaration that specifies a const modifier is a constant enum declaration.
     // In a constant enum declaration, all members must have constant values and
     // it is an error for a member declaration to specify an expression that isn't classified as a constant enum expression.
     
@@ -26,13 +30,19 @@
     const CONST: number = 9000 % 2;
     const enum D {
         d = 10,
         e = 199 * Math.floor(Math.random() * 1000),
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2474: const enum member initializers must be constant expressions.
         f = d - (100 * Math.floor(Math.random() % 8)),
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 !!! error TS2474: const enum member initializers must be constant expressions.
         g = CONST,
+        ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
             ~~~~~
 !!! error TS2474: const enum member initializers must be constant expressions.
     }
\ No newline at end of file
